dev/src/main/resources/META-INF/plugin.xml

```diff
@@ -17,6 +17,15 @@
         Adds support for developing cloud-native, containerized web applications.
     ]]></description>
 
+    <change-notes><![CDATA[
+    <p><b>0.8.0</b></p>
```

Comment:
Hey John, is this supposed to be 0.9.0?  Otherwise, I approve.  (The link is the one Sarah gave us)